components/ui/AltTypography.js

```diff
@@ -54,7 +54,7 @@ H3.defaultProps = { color: 'trueBlack' }
 export const h4Styles = css`
   ${sharedHeaderStyles}
   font-size: 1.8rem;
-  line-height: 2.4rem;
+  line-height: calc(24 / 18);
```

Comment:
no unit (rem or so), is that correct?

Reply:
Yes... line-height is based on the font-size so it can be unit-less. (Best when the font-size changes, so maybe overkill here.)